Droidcon-Boston/app/src/main/java/com/mentalmachines/droidcon_boston/views/detail/AgendaDetailFragment.kt

```diff
@@ -101,8 +101,16 @@ class AgendaDetailFragment : Fragment() {
 
     private fun populateView() {
         tv_agenda_detail_title.text = viewModel.talkTitle
-        tv_agenda_detail_room.text =
+
+        // WORKAROUND FOR https://github.com/Droidcon-Boston/conference-app-android/issues/165
+        // If the talk title is Check-In then hardcode the room name to be the lobby
+        if (viewModel.talkTitle.toLowerCase().contains("check-in")) {
+            tv_agenda_detail_room.text =
+                resources.getString(R.string.str_agenda_detail_room, "Calderwood Pavilion Lobby")
```

Comment:
ditto here.
The primary speaker hack is bad enough in the import scripts.  We don't need all these hardcoded strings in the app as well :-P